src/Orleans/Streams/PubSub/ImplicitStreamSubscriberTable.cs

```diff
@@ -53,7 +51,7 @@ internal void InitImplicitStreamSubscribers(IEnumerable<Type> grainClasses)
             {
                 if (!TypeUtils.IsGrainClass(grainClass))
                 {
-                    return;
+                    continue;
```

Comment:
Throw?  Shouldn't these be grain types?  If type is not a grain, doesn't this indicate there is an error somewhere?